### ISSUE TITLE
Webpack externals and css-modules

### ISSUE THREAD

**Issue by fahad19** [css-modules/css-modules#227]
TL;DR: Need Webpack's `externals`-like feature for css-modules.

## Example: JS-only point of view

With Webpack, we can use [externals](https://webpack.js.org/configuration/externals/) to keep our bundle sizes small, especially when we are loading third-party libraries globally in the browser.

If our `webpack.config.js` had externals configuration for `lodash` like this for example:

```js
// webpack.config.js
module.exports = {
  externals: {
    lodash: '_'
  }
};
```

And our `index.html` happened to be like this:

```html
<script src="/js/lodash.min.js"></script>
<script src="/js/bundle.js"></script>
```

Then all the `require('lodash')` inside `bundle.js` would have been referenced to the global `window._` variable automatically by Webpack.

## Expectation: with css-modules

Now my question is, how can we achieve something similar with css-modules?

Let's assume we want to load [css-skeleton](https://github.com/StevenIseki/css-skeleton) first, and then our custom bundle would be loaded which wants to re-use Skeleton CSS.

```html
<script src="/js/css-skeleton-bundle.js"></script>
<script src="/js/bundle.js"></script>
```

Basically, I don't want `bundle.js` to include any css-skeleton code, since it has already been loaded before.

---

ps: I tried to find something like a mailing list for css-modules but couldn't find one. That's why created an issue.

Thanks!

**Comment by fahad19** [comment]
This looks promising: https://github.com/webpack-contrib/css-loader/pull/177

**Comment by michael-ciniawsky** [comment]
@fahad19 I close this one here and discussion moved to #496 ? In case I missed something, that's only mentioned in the issue/here, feel free to reopen :)
